_opensource/OpCrux.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Operation Crux (OpCrux)
+status: Archive
+language: C++
+image_path: /assets/opcrux_logo.png
+links: [['Website', 'https://opcrux.org', 'fas fa-external-link-alt'], ['Twitter', 'https://twitter.com/opcrux', 'fab fa-twitter'], ['Youtube', 'https://www.youtube.com/channel/UC5-Z2o03q0_HAz-2ntuOfDA', 'fab fa-youtube'], ['GitHub', 'https://github.com/SimonNitzsche/OpCrux-Server', 'fab fa-github']]
```

Comment:
Should we maybe limit this to just the GitHub since the other links arent really relevant any more?

Reply:
With the same reasoning as on the other comment. The other links can be useful to people looking for information about the project, or about LU server project history in general. No harm in keeping the links there as long as it's clear what the recommended project for people to use is.